packit_service/worker/monitoring.py

```diff
@@ -62,9 +62,16 @@ def __init__(self):
             registry=self.registry,
         )
 
-        self.initial_status_time = Histogram(
-            "initial_status_time",
-            "Time it takes to set the initial status",
+        self.first_initial_status_time = Histogram(
+            "first_initial_status_time",
+            "Time it takes to set the initial status for the first check in the first job",
+            registry=self.registry,
+            buckets=(5, 15, 30, float("inf")),
+        )
+
+        self.last_initial_status_time = Histogram(
+            "last_initial_status_time",
+            "Time it takes to set the initial status for the last check in the last job",
             registry=self.registry,
             buckets=(5, 15, 30, float("inf")),
```

Comment:
as we discussed that for the last check, we are ok with a bigger delay, could we add also buckets for 1 minute and 2 minutes to have a better overview?